### PR TITLE
Appveyor:  force install grpcio >= 1.0rc1.

### PR DESCRIPTION
**Pull request by tseaver**
Toward #1863.

### CODE REVIEW

**Comment by tseaver** [comment]
@dhermes I'm going to merge this one as soon as Travis  green, to see if it can un-hork our Appveyor problems.

File: appveyor.yml [hunk 1]
```diff
@@ -82,9 +82,9 @@ install:
   # compiled extensions and are not provided as pre-built wheel packages,
   # pip will build them from source using the MSVC compiler matching the
   # target Python version and architecture
-  - "%CMD_IN_ENV% pip install wheel nose nose-exclude cryptography grpcio"
+  - "%CMD_IN_ENV% pip install wheel nose nose-exclude cryptography"
   # Install sometimes-problemaatic gRPC-related dependencies
```
Comment (dhermes) [inline appveyor.yml]: Can you fix the spelling on "problemaatic" while you're in there?

**Comment by dhermes** [comment]
LGTM. No need to wait on Travis but can you fix the typo I referred to?

PS Thanks for creating the gRPC label.

File: appveyor.yml (hunk 1 above)
Comment (tseaver) [inline appveyor.yml]: Done

**Pull request merged**